source/includes/options-mongotranslate.yaml

```diff
@@ -90,6 +90,26 @@ directive: option
 default: |
   ``test``
 description: |
-  Specifies a database name to use in the output. Defaults
-  to ``test``.
-...
\ No newline at end of file
+  The database name to use with table names in the SQL query if
+  no database is specified. For example:
```

Comment:
Can we write a more complete lead in like:

```
The following example uses ``--dbName`` to query the ``groceries`` database's ``fruit`` 
collection:
```

or something like that.

Reply:
Done.